model/task/db.go

```diff
@@ -2602,7 +2602,16 @@ func getTasksByVersionPipeline(versionID string, opts GetTasksByVersionOptions)
 		{"$project": projectOut},
 		{"$match": match},
 	}
-
+	if !opts.IncludeExecutionTasks {
```

Comment:
Can we prepend it before [this](https://github.com/evergreen-ci/evergreen/blob/04c1e2b8e5df37855c399279853b7b35ae059ba7/model/task/db.go#L2591) as well?

Reply:
Good point, let's leave it at its current placing.